inference-engine/thirdparty/clDNN/src/layout_optimizer.cpp

```diff
@@ -17,6 +17,9 @@
 #include "pooling_inst.h"
 #include "one_hot_inst.h"
 #include "permute_inst.h"
+#include "gemm_inst.h"
+#include "softmax_inst.h"
+#include "gather_inst.h"
```

Comment:
Why we need softmax/gather includes here? Are they used anywhere?

Reply:
not now. I was thinking of applying same thing to gather too because gather is also not supporting the blocked format. However current target does not show that problem in terms of performance, I decided not to apply to them. I will remove them.

Reply:
Seems that there are blocked layout supported fc kernels. I will apply this to only gemm this time?

Reply:
OK